src/main/java/cinema/model/dto/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,34 @@
+package cinema.model.dto;
+
+import cinema.model.Ticket;
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Ticket> tickets;
```

Comment:
it's a bit tricky to work with Tickets (not with Ticket..Dto) - you might get LazyException unless you do some kind of fetching...